### ISSUE TITLE
Change the default settings for MMS [Feature Request]

### ISSUE THREAD

**Issue by lalibi** [klinker-apps/messenger-issues#551]
My contract doesn't include MMS, so on my last bill there were several (rather steep - 0.395€ per MMS) charges because I didn't know my messages would be sent as MMS.

I had a new phone a couple of weeks ago, I installed Pulse, and even though I knew there are a couple of settings for MMS I should take care of, I neglected to set them up upfront.

So, my suggestions are:

1. I had a couple of charges because I sent some long messages that went out as MMS, so I suggest having the default setting of `Convert to MMS` to `Never`.
1. I had a triple charge because I wanted to send the same message to 3 recipients, so  I suggest having the default setting of `Use Group MMS` to `Off`.

In conclusion, I know that all those things can be set up in settings, but I think it's better that a user is "safe" by default, and chose to opt-in for those settings, and not the other way around.

**Comment by klinker24** [comment]
Sorry, I don't see this change happening. 

While I know that some carriers do charge for MMS, the **vast** majority (almost 80%) of Pulse users are in the US, where MMS does not cost extra. With the scale so heavily tipped towards free MMS, I do not want to compromise the usage - or setup - experience, for that majority.

**Comment by lalibi** [comment]
Fair enough, but please consider one of the two following alternatives.

1. The default values for those two options remain the same, but on first run present the user with a screen that asks if he/she'd like to turn those off, with a note like "if you are charged extra for MMS, consider turning those off, otherwise keep the settings as is"
1. Not on the first run, but rather on the first time someone tries to send a long or a group SMS, present them with a warning (with a "never show this again" option)", stating that extra charges may occur, including a way to the change the corresponding option.

I think the 2nd option especially is rather rational (ux-wise), I hope you can justify the resources needed to implement it.

**Comment by Skullmonkey** [comment]
He has already stated that he does not want to create a setup process. I do believe that this is one of the global settings that he could sync between devices like I have requested in #498 though. This way, when you log in to your account and the settings get synced, It would functionally be the same as you have requested with no user interactions from all parties.

**Comment by lalibi** [comment]
I don't think the second option compromises the setup process.

It seems the developer is very strong minded on this one, so there is no point discussing it further. But, I have to say that the possibility that, up to 20% of your users may get charged because the app switches from SMS to MMS implicitly, is not something to be taken lightly. 

This is just my honest take on the matter, I'm not trying to get bitter.
